README.md

```diff
@@ -300,6 +286,16 @@ In config page template
 {{/tableSections}}
 ```
 
+This assumes all steps containing fields have a `section` - `locals.section`, this is used to group fields in the confirm table and the email.
```

Comment:
I just thought that perhaps you could make more of a point about it and a little code example.